python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -68,24 +71,23 @@ class LatissCWFSAlign(salobj.BaseScript):
     -----
     **Checkpoints**
 
-    * intra: when taking intra focus image.
-    * extra: when taking extra focus image.
-    * cwfs: when running CWFS code.
-
+    None
+    
     **Details**
 
     This script is used to perform measurements of the wavefront error, then
     propose hexapod offsets based on an input sensitivity matrix to minimize
-    the errors.
+    the errors. The hexapod offsets are not applied automatically and must be
+    performed by the user.
 
     """
 
     __test__ = False  # stop pytest from warning that this is not a test
 
-    def __init__(self, index, remotes=True):
+    def __init__(self, index=1, remotes=False):
 
         super().__init__(
-            index=index, descr="Perform optical alignment procedure on LATISS data."
+            index=index, descr="Perform optical alignment procedure on LATISS in-focus images using Curvature-Wavefront Sensing Techniques."
```

Comment:
Are you sure "in-focus image" is the correct term here? Maybe just

>Perform optical alignment procedure on LATISS images using Curvature-Wavefront Sensing Techniques.

Reply:
Changed to: 
Perform optical alignment procedure of the Rubin Auxiliary Telescope with LATISS using Curvature-Wavefront Sensing Techniques